cmd/machine-api-migration/main.go

```diff
@@ -165,8 +204,19 @@ func main() {
 		os.Exit(1)
 	}
 
-	klog.Info("starting manager")
+	reconciler := machinesync.MachineSyncReconciler{
+		Platform: provider,
+
+		MAPINamespace: *mapiManagedNamespace,
+		CAPINamespace: *managedNamespace,
```

Comment:
I wonder if the capi managed namespace should have capi in the name at the flag level to match?